src/parser.ts

```diff
@@ -440,7 +435,7 @@ export function parseFromProgram(
 					return t.elementNode('elementType');
 				}
 				case 'React.ReactNode': {
-					return t.unionNode([t.elementNode('node'), t.undefinedNode()]);
+					return t.unionNode([t.undefinedNode(), t.elementNode('node')]);
```

Comment:
Using the convention that `undefinedNode` always comes first in a `unionNode` makes it easy to compare the expected to actual objects in the tests.
This was the quick fix instead of creating a matcher for jest that would ignore array order within union nodes.
It's also why I had to change a couple of the `output.json` test files to reorder elements.